scripts/background.js

```diff
@@ -3,7 +3,7 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
   if (
     changeInfo.status === 'complete' &&
     tabUrl &&
-    (tabUrl.includes('linkedin.com/feed') || tabUrl.includes('linkedin.com/in'))
+    (tabUrl.includes('youtube.com/results'))
```

Comment:
When a user is on this particular url (the url for search results), the global.css file will be injected and overwrite the existing css styles for the specified css selectors.